src/Symfony/Component/Form/Extension/Core/DataTransformer/PercentToLocalizedStringTransformer.php

```diff
@@ -52,12 +103,17 @@ public function __construct(int $scale = null, string $type = null)
             $type = self::FRACTIONAL;
         }
 
+        if (null === $roundingMode) {
+            $roundingMode = self::ROUND_HALF_UP;
+        }
+
         if (!\in_array($type, self::$types, true)) {
             throw new UnexpectedTypeException($type, implode('", "', self::$types));
         }
 
         $this->type = $type;
         $this->scale = $scale;
+        $this->roundingMode = $roundingMode;
```

Comment:
`$this->roundingMode = $roundingMode ?? self::ROUND_HALF_UP;` would be enough

Reply:
I made the code consistent with the type, scale and others Transformers. Isn't better ?